concepts/05 UI Components/TreeList/50 Selection/20 API/1 Initial and Runtime Selection.md

```diff
@@ -91,6 +167,87 @@ You can select rows at runtime using the [selectRows(keys, preserve)](/api-refer
         ],
         // ...
     })
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxTreeList ...
+            v-model:selected-row-keys="selectedRowKeys"
+            @content-ready="selectFirstRow">
+        </DxTreeList>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxTreeList from 'devextreme-vue/tree-list';
+
+    export default {
+        components: {
+            DxTreeList
+        },
+        data() {
+            return {
+                selectedRowKeys: []
+            }
+        },
+        methods: {
+            selectFirstRow(e) {
+                const rowKey = e.component.getKeyByRowIndex(0);
+                this.selectedRowKeys.push(rowKey);
+            }
```

Comment:
This approach will not work in V3 with our components due to the breaking change. It will work only for ordinary "v-for". The Platofrms squad will research this issue. For this case, use the reactive approach:
```js
this.selectedRowKeys = [...this.selectedRowKeys, rowKey];
```